### PR TITLE
Rpm installation for superset

### PR DESCRIPTION
**Pull request by ankursinghal2005**
RPM based installation

**Installation Guide-:** 

https://guavus.atlassian.net/wiki/spaces/CU/pages/71368884/Installation+Guide+for+Apache+Superset

### CODE REVIEW

**Comment by ankursinghal2005** [comment]
You want to replace superset-solution-installer with superset-installer every where as package name

**Comment by arpit-agarwal** [comment]
> You want to replace superset-solution-installer with superset-installer every where as package name

Yes

File: rpm-mgmt/artifactory.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+export ARTIFACTORY_USERNAME=dev-deployer
+export ARTIFACTORY_PASSWORD=dev@guavus
```
Comment (bipinsoniguavus) [inline rpm-mgmt/artifactory.sh]: is there a only way to keep these credentials in `sh` file?

File: rpm-mgmt/artifactory.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+export version=`cat ../VERSION`
+
+MAJOR_VERSION=$(echo $version | awk -F. '{print $1}')
+MINOR_VERSION=$(echo $version | awk -F. '{print $2}')
+PATCH_VERSION=$(echo $version | awk -F. '{print $3}')
+
+export RPM_ARTIFACTORY=artifacts.ggn.in.guavus.com:4245/ggn-dev-rpms/guavus/superset/release/
```
Comment (bipinsoniguavus) [inline rpm-mgmt/artifactory.sh]: can we generalize this in a way that it could read the artifactory path from somewhere else?

**Pull request merged**